src/components/Game/Game.js

```diff
@@ -131,9 +127,10 @@ class GameView extends Component {
                 <Backspace
                     clicked={this.handleBackspaceClick}
                     areSlotsEmpty={lettersInSlots.length === 0}
+                    isSubmitVisible={isSubmitVisible}
                 />
                 {
-                    !answer ? lettersOrSubmit : <Result />
+                    lettersOrSubmit
```

Comment:
maybe it's a matter of preference but I would rather inline the value of `lettersOrSubmit` here (getting rid of `lettersOrSubmit` variable)
I would say that if you want split things out then consider creating a subcomponent

Reply:
👍